client/src/components/forms/CommentForm.js

```diff
@@ -40,7 +41,7 @@ export default function CommentForm(props) {
         placeholder="Leave your feedback"
         ref={register({ required: true })}
       ></textarea>
-      {errors["input-feedback"] && <span>This field is required</span>}
+      <FormError error={errors["input-feedback"]} errorMsg={"This field is required"} />
```

Comment:
i think it's better to have the checking done outside of the component so the app doesnt need to render every FormError component every time. But then we kind run into the problem that this component becomes very similar to what we had previously and possibly redundant?

Reply:
I feel like it's redundant as well. Form error doesn't really need its own component imo, I liked how it was before.